src/main/java/com/minecolonies/core/generation/defaults/DefaultRecipeProvider.java

```diff
@@ -637,6 +637,16 @@ private void buildOtherItems(@NotNull final Consumer<FinishedRecipe> consumer)
                 .unlockedBy("has_boat", has(ItemTags.BOATS))
                 .save(consumer);
 
+        ShapedRecipeBuilder.shaped(RecipeCategory.MISC, ModItems.large_empty_bottle, 4)
+          .pattern("PSP")
+          .pattern("P P")
+          .pattern("PGP")
+          .define('P', Items.GLASS_PANE)
+          .define('G', Items.GLASS)
+          .define('S', ItemTags.WOODEN_SLABS)

```

Comment:
Maybe a wooden button?  A slab seems a bit big for a bottle, even if you are making 4 at a time.
Though the slab is (oddly) cheaper, so maybe this makes sense.

Reply:
Also buttons are mechanic items, so they're more expensive that way too.

Reply:
Yeah,  I wanted to make it a button, but in the crafting grid the slab looks nicer